### PR TITLE
Clustal

### PR DESCRIPTION
**Pull request by JensTimmerman**

### CODE REVIEW

**Comment by JensTimmerman** [comment]
jenkins: test this please

**Comment by JensTimmerman** [comment]
jenkins: please test this

**Comment by JensTimmerman** [comment]
jenkins test the latest commit please

**Comment by JensTimmerman** [comment]
jenkins, test this latest commit please

File: easybuild/easyconfigs/a/argtable/argtable-2.13-ictce-5.5.0.eb [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+name = 'argtable'
+version = '2.13'
+
+homepage = 'http://argtable.sourceforge.net/'
+description = """ Argtable is an ANSI C library for parsing GNU style
+ command line options with a minimum of fuss. """
+toolchain = {'name': 'ictce', 'version': '5.5.0'}
+
+source_urls = [SOURCEFORGE_SOURCE]
+sources = ['%(name)s%s' % version.replace('.', '-)]
```
Comment (boegel) [inline easybuild/easyconfigs/a/argtable/argtable-2.13-ictce-5.5.0.eb]: you'll need to escape: `%%(name)s`

File: easybuild/easyconfigs/c/Clustal-Omega/Clustal-Omega-1.2.0-ictce-5.5.0.eb [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+version = '1.2.0'
+
+homepage = 'http://www.clustal.org/omega/'
+description = """ Clustal Omega is a multiple sequence alignment
+ program for proteins. It produces biologically meaningful multiple
+ sequence alignments of divergent sequences. Evolutionary relationships
+ can be seen via viewing Cladograms or Phylograms """
+toolchain = {'name': 'ictce', 'version': '5.5.0'}
+
+dependencies = [('argtable', '2.13')]
```
Comment (boegel) [inline easybuild/easyconfigs/c/Clustal-Omega/Clustal-Omega-1.2.0-ictce-5.5.0.eb]: deps below sources

File: easybuild/easyconfigs/c/Clustal-Omega/Clustal-Omega-1.2.0-ictce-5.5.0.eb [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+
+name = 'Clustal-Omega'
+version = '1.2.0'
+
+homepage = 'http://www.clustal.org/omega/'
+description = """ Clustal Omega is a multiple sequence alignment
+ program for proteins. It produces biologically meaningful multiple
+ sequence alignments of divergent sequences. Evolutionary relationships
+ can be seen via viewing Cladograms or Phylograms """
+toolchain = {'name': 'ictce', 'version': '5.5.0'}
```
Comment (boegel) [inline easybuild/easyconfigs/c/Clustal-Omega/Clustal-Omega-1.2.0-ictce-5.5.0.eb]: empty line above

**Comment by boegel** [comment]
Jenkins: test this please

File: easybuild/easyconfigs/a/argtable/argtable-2.13-ictce-5.5.0.eb [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+name = 'argtable'
+version = '2.13'
+
+homepage = 'http://argtable.sourceforge.net/'
+description = """ Argtable is an ANSI C library for parsing GNU style
+ command line options with a minimum of fuss. """
+toolchain = {'name': 'ictce', 'version': '5.5.0'}
+
+source_urls = [SOURCEFORGE_SOURCE]
+sources = ['%%(name)s%s' % version.replace('.', '-)]
```
Comment (boegel) [inline easybuild/easyconfigs/a/argtable/argtable-2.13-ictce-5.5.0.eb]: missing `'` at the end

**Comment by migueldiascosta** [comment]
@JensTimmerman, this PR is being closed for the following reason(s): no activity for > 6 months.
Please don't hesitate to reopen this PR or add a comment if you feel this contribution is still relevant.
For more information on our policy w.r.t. closing PRs, see https://easybuild.readthedocs.io/en/latest/Contributing.html#why-a-pull-request-may-be-closed-by-a-maintainer

**Pull request closed without merging**